applications.yaml

```diff
@@ -702,9 +710,14 @@
   identifier:
     kind: Class
     id: TeamsWebView
-    matching_strategy: Legacy
+    matching_strategy: Equals
   options:
   - tray_and_multi_window
+  - object_name_change
```

Comment:
I don't think `object_name_change` is necessary here

Reply:
I am having the same issue as specified in [bug#652](https://github.com/LGUG2Z/komorebi/issues/652) and adding ```object_name_change``` solved it for me.  I agree with @tardyp , I think this comes down to a difference in systems.  I am running an older version of Windows 10 Enterprise (Version 10.0.19044, to be exact).

If adding ```object_name_change``` does not have any negative effects for other users, I think this change should be pushed through to main.